aws/resource_aws_codebuild_project_test.go

```diff
@@ -686,3 +737,40 @@ resource "aws_codebuild_project" "foo" {
 }
 `, rName, authResource, authType)
 }
+
+func hclVpcResources() string {
```

Comment:
FYI these function names are global across the entire provider so we would generally prefer sticking with the long for `testAccAWSCodeBuildProjectConfig_X`